sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/nlp/automl_nlp_job.py

```diff
@@ -25,6 +25,38 @@
 
 # pylint: disable=too-many-instance-attributes,protected-access
 class AutoMLNLPJob(AutoMLVertical, ABC):
```

Comment:
Could you add an inline example for this?

Reply:
This class is not to be instantiated by users and is not visible in the documentation. So, I won't include an example for the constructor.

Reply:
That's my bad for missing the "You should not instantiate..." notice in the docstring :)